DogeNet.IdentityServer/Program.cs

```diff
@@ -0,0 +1,45 @@
+// <copyright file="Program.cs" company="Leobit">
+// Copyright (c) Leobit. All rights reserved.
+// </copyright>
+
+namespace DogeNet.IdentityServer
+{
+    using System;
+    using DogeNet.DAL;
+    using Microsoft.AspNetCore.Hosting;
+    using Microsoft.Extensions.DependencyInjection;
+    using Microsoft.Extensions.Hosting;
+    using Microsoft.Extensions.Logging;
+
+    public class Program
+    {
+        public static void Main(string[] args)
+        {
+            var host = CreateHostBuilder(args).Build();
+
+            using (var scope = host.Services.CreateScope())
```

Comment:
We should make this on our main project level (web)

Reply:
Ok